.github/workflows/tests.yml

```diff
@@ -0,0 +1,61 @@
+name: Tests
+on:
+  pull_request:
+  push:
+    branches:
+    - master
+
+jobs:
+  track-config:
+    name: Check track configuration
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Fetch
+        run: bin/fetch-configlet
+      - name: Lint
+        run: bin/configlet lint .
+      - name: Check for invalid UUIDs
+        # can be removed once `configlet lint` gains this ability.
+        # Check issue https://github.com/exercism/configlet/issues/99
+        run: |
+          uuids=$(jq --raw-output '.exercises | map(.uuid) | .[]' config.json)
+          bad_uuid=$(echo "$uuids" | grep -vE '^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$' || true)
```

Comment:
note: GHA runs this with `bash -e`, and `grep` with no matches exits 1. Since this `grep` is looking for bad UUIDs, this makes it so that ironically if there are no bad UUIDs, the whole thing exits 1 (because of `-e`)! Therefore, I had to put `|| true` in.